pennylane_lightning/core/_serialize.py

```diff
@@ -285,6 +285,9 @@ def _pauli_word(self, observable, wires_map: dict = None):
         def map_wire(wire: int):
             return wires_map[wire] if wires_map else wire
 
+        if len(observable) == 0:
```

Comment:
```suggestion
        if observable is None:
```

Reply:
Tried but failed. Will keep previous implementation.

Reply:
@maliasadi this won't work because `PauliWord` is a subclass of `OrderedDict`, and if the only operator inside it is `Identity`, it automatically removes it as it's redundant.

Reply:
Haa... thanks @mudit2812!

Reply:
```suggestion
        if not observable:
```